package.json

```diff
@@ -9,7 +9,7 @@
     "angular-animate": "^1.5.0",
     "angular-aria": "^1.5.0",
     "angular-marked": "^1.2.2",
-    "angular-material": "~1.0.9",
+    "angular-material": "^1.0.7",
```

Comment:
Since apparently angular-material makes breaking changes in minor versions, we should probably still lock this down, e.g. `~1.1.0`